storage/statedb/sync.go

```diff
@@ -294,7 +294,7 @@ func (s *TrieSync) schedule(req *request) {
 	s.retrievedByDepth[req.depth]++
 
 	// Schedule the request for future retrieval
-	s.queue.Push(req.hash, int64(req.depth))
+	s.queue.Push(req.hash, req.depth)
```

Comment:
Previous codes insert the value as `int64`, but now it is `int`, is it intended?